.github/workflow_templates/e2e-daily.yml

```diff
@@ -0,0 +1,51 @@
+{!{- $enableWorkflowOnTestRepos := false -}!}
+{!{- $workflowName := "Daily e2e tests" -}!}
+name: '{!{ $workflowName }!}'
+on:
+  schedule:
+  - cron: '5 21 * * *'
+  workflow_dispatch:
+
+env:
+  WERF_DRY_RUN: "false"
+{!{ tmpl.Exec "werf_envs" | strings.Indent 2 }!}
+
+# Always run a single job at a time.
+# Note: Concurrency is currently in beta and subject to change.
+# https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#concurrency
+concurrency:
+  group: e2e-daily
+
+jobs:
+  skip_tests_repos:
+    name: Skip tests repos
+    runs-on: ubuntu-latest
+    if: ${{ {!{ $enableWorkflowOnTestRepos }!} || github.repository == 'deckhouse/deckhouse' }}
```

Comment:
It's ok to skip the workflow in this way and it looks "skipped" in Actions list.

It's not a request for changes, just a note. This way you can't return a notice to user: "why it was skipped?". See this [run](https://github.com/deckhouse/deckhouse-test-1/actions/runs/2690156391) for an example.

Reply:
ok, i got it, but after 30 minutes i had finished research and did not added notice. 

It is difficult here, because we need to base from js image with actions library. It think, it is not important.